### PR TITLE
[vcpkg] add env var VCPKG_OVERLAY_TRIPLETS

### PR DESCRIPTION
**Pull request by Neumann-A**
logical continuation of #12640

### CODE REVIEW

**Review by JackBoosY — APPROVED** [review]

File: docs/users/config-environment.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ the platform dependent PATH seperator (Windows `;` | others `:`)
 
 Example (Windows): `C:\custom-ports\boost;C:\custom-ports\sqlite3`
 
+#### VCPKG_OVERLAY_TRIPLETS
+
+This environment variable allows users to add directories to search for triplets.
+[Example: overlay triplets](../examples/overlay-triplets-linux-dynamic.md).
+List paths to overlays using the platform dependent PATH seperator (Windows `;` | others `:`)
```
Comment (ras0219) [inline docs/users/config-environment.md:53]:
```suggestion
List paths to overlays using the platform dependent PATH seperator (Windows `;`, others `:`) 
```

**Review by ras0219 — APPROVED** [review]

File: toolsrc/src/vcpkg/base/strings.cpp [hunk 2]
```diff
@@ -182,6 +182,15 @@ std::vector<std::string> Strings::split(StringView s, const char delimiter)
     }
 }
 
+std::vector<std::string> Strings::split_paths(StringView s)
+{
+#if defined(_WIN32)
+    return Strings::split(std::move(s), ';');
+#else // ^^^ defined(_WIN32) // !defined(_WIN32) vvv
+    return Strings::split(std::move(s), ':');
```
Comment (strega-nil) [inline toolsrc/src/vcpkg/base/strings.cpp:190]:
```suggestion
    return Strings::split(s, ';');
#else // ^^^ defined(_WIN32) // !defined(_WIN32) vvv
    return Strings::split(s, ':');
```

`s` is a `StringView`, there's no point to `move`-ing it.

**Review by strega-nil — APPROVED** [review]
Just the one change :)

**Comment by JackBoosY** [comment]
@BillyONeal Could you please look at the failure logs on the pipeline test?

**Comment by JackBoosY** [comment]
```
A suitable version of git was not found (required v2.7.4) and unable to automatically download a portable one. Please install a newer version of git.
```

**Comment by oswjk** [comment]
Is this feature planned to land at master at some point?

**Comment by ras0219-msft** [comment]
Thanks!

**Pull request merged**